doc/development/deprecations.rst

```diff
@@ -44,7 +44,7 @@ Pending deprecations
   - Behaviour will change in v0.33
 
 * The public methods of ``DefaultQubit`` are pending changes to
-  follow the new device API, as used in ``DefaultQubit2``.
+  follow the new device API.
```

Comment:
Should we view this deprecation as complete?

Reply:
I'd rather make it official when switching the entrypoint because that's how most people get it (but it will be soon)